### PR TITLE
fix: type for vite config

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/vite.config.ts [hunk 1]
```diff
@@ -61,6 +60,7 @@ export default defineConfig({
     },
     setupFiles: ['tests/unit/setup-files/setup.ts'],
     coverage: {
+      provider: 'istanbul',
```
Comment (kelsos) [inline frontend/app/vite.config.ts:63]: why not c8? also maybe it makes sense to split vitest + vite config?

File: frontend/app/package.json [hunk 2]
```diff
@@ -79,9 +79,8 @@
     "@types/lodash": "4.14.195",
     "@types/node": "18.16.16",
     "@vitejs/plugin-vue2": "2.2.0",
-    "@vitest/coverage-c8": "0.31.4",
+    "@vitest/coverage-v8": "0.32.2",
     "@vue/test-utils": "1.3.5",
-    "c8": "7.14.0",
```
Comment (lukicenturi) [inline frontend/app/package.json:84]: OOT, is this needed?

File: frontend/app/vite.config.ts (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/vite.config.ts]: Sorry I saw the wrong thing, Istanbul is for e2e.

File: frontend/app/package.json (hunk 2 above)
Comment (kelsos) [reply frontend/app/package.json:84]: If things pass then we are safe right? 😄

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**